modules/authorization/con-permission-policy-and-role-source.adoc

```diff
@@ -0,0 +1,29 @@
+[id='con-permission-policy-and-role-source']
+= Understanding permission policy and role configuration source
+
+You can manipulate permission policies and roles based on the following designated source information:
+
+* CSV file
+* Configuration file
+* REST API
+* Legacy
+
+Each permission policy and role is associated with a source to maintain data consistency.
+Therefore, when a policy has been created in a CSV file, you cannot change it by using the REST API.
+
+
+Managing roles and permission policies originating from CSV files and REST API involves straightforward modification based on their initial source information.
+
+The Configuration file pertains to the default `role:default/rbac_admin` role provided by the RBAC plugin.
+The default role has limited permissions to create, read, update, and delete permission policies or roles, and to read catalog entities.
+
+[NOTE]
+====
+In case the default permissions are insufficient for your administrative requirements, you can create a custom admin role with required permission policies.
```

Comment:
We can also provide information that our RBAC backend supports a super admin option in the configuration (rbac.admin.superUsers).

Reply:
This content will need further update. I'll merge it as it is and rework it later, in a dedicated PR.